source/mouseHandler.py

```diff
@@ -3,6 +3,11 @@
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+"""
+When working on this file, consider moving to winAPI.

```

Comment:
Let's leave these comments out for now, it isn't clear where the line is for what is "generic Windows abstraction" vs core NVDA behavior. Many files probably have these two concepts coupled, and it will require a careful approach to extract the generic Windows abstraction part into winAPI.